### DISCUSSION TITLE
Replication issue : missing relations

### DISCUSSION THREAD

**Discussion by renaudk** [osm-search/Nominatim#2725] [Q&A]
**Describe the bug**
Our nominatim installation says it's up to date, no errors, no warning but we have missing relations.

**To Reproduce**
Hard to say, we built nominatim servers last year on v3.7.2. Since, replication is running fine. This happens on our 3 servers.
Example:
https://nominatim.openstreetmap.org/ui/details.html?osmtype=R&osmid=45155&class=boundary
This relation is missing on our nodes:
> No place with that OSM ID found. 

We have planned a rebuild of the three nodes (full planet import) but is there a way to check database integrity? At least we need to be informed when the replication fails something.

**Software Environment (please complete the following information):**
- Nominatim version: 3.7.2
- Postgresql version:  12
- Postgis version: ?
- OS: ubuntu 20.04

**Hardware Configuration (please complete the following information):**
- RAM: 32 GB
- number of CPUs: 1 6c/12t
- type and size of disks: 2x512GB NVME RAID-0 (ZFS)
- bare metal/AWS/other cloud service: bare metal (OVH)

**Postgresql Configuration:**
Only recommended changes

**Additional context**
Nothing

**Comment by lonvia** [comment]
This is an old version of Nominatim which has been running updates for a year. There is no way to debug this from afar.

I'll move this to discussions. Maybe someone has a similar setup and knows more. But it is not something actionable for the issue tracker.

**Comment by Stainless5792** [comment]


**Reply by mtmail** [reply]
Please submit as new discussion. Lists the version numbers, date of import, examples what you believe is missing.
